pkgs/toolbox/toolbox.sh

```diff
@@ -162,6 +162,7 @@ Terraform related commands:
  make-terraform13-shell provider [provider...]  -- create a terraform 0.13 shell with the specified providers
  make-terraform14-shell provider [provider...]  -- create a terraform 0.14 shell with the specified providers
  make-terraform15-shell provider [provider...]  -- create a terraform 0.15 shell with the specified providers
+ make-terraform_1_0_0-shell provider [provider...]  -- create a terraform 1.0 shell with the specified providers
```

Comment:
Since terraform is now stable, we could expose a `make-terraform-shell` subcommand that will use the latest stable release ?

Reply:
Old release should be deprecated